samples/boards/argonkey/README.rst

```diff
@@ -0,0 +1,74 @@
+.. _ArgonKey:
+
+ArgonKey sensor board
+#####################
+
+Overview
+********
+This sample provides an example of how to read sensor data
+from the ArgonKey board. The result is displayed on the console.
+It makes use of both the trigger and poll methods.
+
+Requirements
+************
+
+This sample just requires the ArgonKey board.
+The user may select or unselect the sensors from
+:file:`samples/boards/argonkey/prj.conf`.
+
+Please note that all sensor related code is conditionally compiled
+using the `#ifdef` directive, so this sample is supposed to always
+build correctly. Example:
+
+ .. code-block:: c
+
+  #ifdef CONFIG_HTS221
+    struct device *hum_dev = device_get_binding("HTS221");
+
+    if (!hum_dev) {
+      printk("Could not get pointer to %s sensor\n", "HTS221");
+      return;
+    }
+  #endif
+
+References
+**********
+
+ - ArgonKey board.
```

Comment:
List item should not be indented (remove leading space before the ``-``).   This should probably be a link to our ArgonKey board documentation, so change to:

    - :ref:`96b_argonkey`

Reply:
OK.
Do I have to change also the list:
 #. *LPS22HB* baro/temp
 #. *HTS221* humidity
 #. *LSM6DSL* accel
 #. *LSM6DSL* gyro
 #. *LIS2MDL* magnetometer (attached to *LSM6DSL*)

Correct?

Reply:
Well, I just did it!